style-guide/about-style-guide.md

```diff
@@ -0,0 +1,183 @@
+# The style guide
+
+The style guide template itself occasionally includes placeholder text that
+is intended for you to replace with your own text. This text is indicated in
+{curly brackets}.
+
+## When do I need a style guide?
+
+A style guide provides project contributors with general guidelines for writing
+project documentation. The overall goal of a style guide is to ensure quality
+and consistency throughout the project’s documentation, which is especially
```

Comment:
It looks like you have smart apostrophes in some parts, and straight in others. I would recommend a find and replace to make them all straight, just so you don't run into problems with displaying the smart ones later on.

Reply:
A very good catch. If we have the bandwidth, it might be nice to install some kind of pre-commit check for this. I've opened an issue for consideration: https://github.com/thegooddocsproject/templates/issues/159